docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -1,10 +1,10 @@
 ---
 productId: x-date-pickers
-title: Date and Time Pickers - Overriding component structure
+title: Date and Time Pickers - Custom slots and subcomponents
 components: DateTimePickerTabs, PickersActionBar, DatePickerToolbar, TimePickerToolbar, DateTimePickerToolbar, PickersCalendarHeader, PickersShortcuts
 ---
 
-# Overriding component structure
+# Custom slots and subcomponents
 
 <p class="description">The date picker lets you customize subcomponents.</p>
```

Comment:
Date Pickers should be capitalized and plural - here's a suggestion to make this description more "actionable."

```suggestion
Learn how to override the default DOM structure of the Date Pickers.
```

Reply:
Or make it `Date and Time Pickers` in such case to be more correct. 🙈